osmoutils/cosmwasm/helpers.go

```diff
@@ -118,11 +121,25 @@ func Sudo[T any, K any](ctx sdk.Context, contractKeeper ContractKeeper, contract
 		return response, err
 	}
 
-	responseBz, err := contractKeeper.Sudo(ctx, sdk.MustAccAddressFromBech32(contractAddress), bz)
+	// Defer to catch panics in case the sudo call runs out of gas.
+	defer func() {
+		if r := recover(); r != nil {
+			response = *new(K) // Create an empty version of response type K
```

Comment:
Can we have the response to nil instead?

Reply:
Nil isn't possible since `K` is a generic but I did update this to properly fall back to generic defaults 114ca628b25d4806d2262eb8965aecf4680270e9